core/node/node.go

```diff
@@ -45,42 +44,33 @@ import (
 )
 
 // NewNode function creates new Mysterium node by given options
-func NewNode(options Options) *Node {
-	networkDefinition := GetNetworkDefinition(options.NetworkOptions)
-	mysteriumClient := server.NewClient(networkDefinition.DiscoveryAPIAddress)
-
+func NewNode(
```

Comment:
We could use NodeOption or NodeFlags structure here, not multi parameter factory.. as per: https://hackmongo.com/page/golang-antipatterns/#runaway-arguments

Reply:
Yeah, that's what i'm moving towards. But not so easy while we have 3 executables. Would like to do it in upcoming PR